app/controllers/financialOverview/psa/SelectPenaltiesQuarterController.scala

```diff
@@ -70,24 +67,20 @@ class SelectPenaltiesQuarterController @Inject()(
             "submitUrl" -> routes.SelectPenaltiesQuarterController.onSubmit(year).url,
             "year" -> year
           )
-
           renderer.render(template = "financialOverview/psa/selectQuarter.njk", json).map(Ok(_))
         } else {
           Future.successful(Redirect(controllers.routes.SessionExpiredController.onPageLoad))
         }
-
     }
   }
 
-  def onSubmit(year: String, journeyType: ChargeDetailsFilter): Action[AnyContent] = identify.async { implicit request =>
+  def onSubmit(year: String, journeyType: ChargeDetailsFilter = All): Action[AnyContent] = identify.async { implicit request =>
     psaPenaltiesAndChargesService.getPenaltiesForJourney(request.psaIdOrException.id, journeyType).flatMap { penaltiesCache =>
```

Comment:
cannot we pass this All from app.routes. I guess that's generally how we have done for other part of the code

Reply:
It was not needed, already setting in app.routes, removed.